sdk/python/kfp/compiler_cli_tests/test_data/lightweight_python_functions_v2_pipeline-preprocess.yaml

```diff
@@ -0,0 +1,114 @@
+components:
```

Comment:
Why do we have all these as additions?

Reply:
Sorry, didn't read the full file name. Instead of having a snapshot for each component used in our pipelines--many of which are duplicate--can we test some typical components alone, like a few dedicated files contain only components? This way we are not only certain on what we're testing, but also show that components aren't necessarily affiliate to pipelines at compile time. WDYT?

Reply:
Yes, definitely. Good thought on the dups. I do think it's helpful to include a broad range of components, however, in an effort to catch any accidental compiler regressions. I will update.